src/librbd/api/Namespace.cc

```diff
@@ -20,6 +20,7 @@ int Namespace<I>::create(librados::IoCtx& io_ctx, const std::string& name)
   ldout(cct, 5) << "name=" << name << dendl;
 
   if (name.empty()) {
+    lderr(cct) << "namespace name was not specified" << dendl;
```

Comment:
Since the rbd CLI already catches this possibility [1], how did you encounter this issue?

[1] https://github.com/ceph/ceph/blob/master/src/tools/rbd/action/Namespace.cc#L36

Reply:
hi, dillaman. I don't test this issue, and I add this log info when I analyse the code. If it's superfluous, I will close this commit. thanks a lot.